AssistPanel: fix for onclick handler and fix for example

TomaszLukawskiSam opened the pull request:
[Issue] SAD-802
[Problem] Text is cut in example
[Solution]
 - Text has been modified
 - Fix for onclick handler (wrong binding)

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

lslachciak reviewed (commented):
please separate fixes

TomaszLukawskiSam committed:
(wearable) AssistPanel: fix for text content in example

[Issue] SAD-802
[Problem] Text is cut in example
[Solution]
  - Text has been modified

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

TomaszLukawskiSam committed:
(wearable) AssistPanel: fix for onclick handler

[Issue] N/A
[Problem] Click on Assist Panel button doesn't work
[Solution]
  - Fix for onclick handler (wrong binding)

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>
